lib/engine/ability/blocks_division.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
+  module Ability
+    class BlocksDivision < Base
```

Comment:
This isn't intuitive to what 'Division' means

Reply:
Add an overview to lib/engine/ability/README.md

Reply:
Would suggest 'Partition' as this tends to relate separation of geographical things.

And BlocksCrossingPartitions would make it clear what the ability does.

Reply:
Otherwise good from my pov.

Reply:
All done :)